sympy/functions/special/tests/test_error_functions.py

```diff
@@ -48,7 +48,7 @@ def test_erf():
     assert erf(x).as_leading_term(x) == 2*x/sqrt(pi)
     assert erf(x*y).as_leading_term(y) == 2*x*y/sqrt(pi)
     assert (erf(x*y)/erf(y)).as_leading_term(y) == x
-    assert erf(1/x).as_leading_term(x) == S.One
+    assert erf(1/x).as_leading_term(x) == erf(1/x)
```

Comment:
`erf` has finite limits `+/-1` at `+/-oo` (unlike `erfi`) so this test need not be changed.